types/chrome/index.d.ts

```diff
@@ -1194,6 +1194,7 @@ declare namespace chrome.contextMenus {
         /** Optional. Note: You cannot change an item to be a child of one of its own descendants.  */
         parentId?: any;
         type?: string;
+        visible?: boolean;
```

Comment:
How about adding [tsdoc](https://github.com/microsoft/tsdoc) comment with `@since`?
The document you gave says this property is since Chrome 62.

Try to write like this
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/e5fcac4bf0d1706e4a3cc64fc1ff84e78184b59b/types/chrome/index.d.ts#L1297L1300

Reply:
Added the comment as per your suggestion using the format suggested, similar to the `visible` property above in `createProperties`.